copter/source/docs/acro-mode.rst

```diff
@@ -8,7 +8,8 @@ Acro mode uses the RC sticks to control the angular velocity
 of the copter in each axis. Release the sticks and the vehicle will maintain its
 current attitude and will not return to level (attitude hold). Acro mode is useful for
 aerobatics such as flips or rolls, or FPV when smooth and fast control
-is desired.
+is desired. It's generally a good idea to dial down the gyro low-pass filter and compensate
```

Comment:
```suggestion
is desired. Raising the :ref:`INS_GYRO_FILTER<INS_GYRO_FILTER>` cutoff frequency and retuning may allow better responsiveness, but appropriate harmonic notch filtering to prevent noise impacts must be setup prior to retuning. 
```

Reply:
Agree